apps/bmd/src/App.test.tsx

```diff
@@ -2,8 +2,10 @@ import * as React from 'react'
 import { render } from '../test/testUtils'
 import App from './App'
 import { fakeMachineConfigProvider } from '../test/helpers/fakeMachineConfig'
+import { advanceTimersAndPromises } from '../test/helpers/smartcards'
```

Comment:
fyi this exists in utils now too https://github.com/votingworks/vxsuite/blob/main/libs/test-utils/src/advanceTimers.ts